Python/README.md

```diff
@@ -203,7 +203,11 @@ _add list here_
 - [Maximum Square Matrix with all 1's](other/Maximum_Square_Matrix.py)
 - [Reverse a string using stack](other/Reverse_string_using_stack.py)
 - [Russian Peasant Algorithm](other/Russian_Peasant_Algorithm.py)
+- [Satellite Image Analysis using numpy](other/Satellite_Image_Analysis_using_numpy.ipynb)
+- [Swap Two Nibbles](other/Swap_two_nibbles.py)
+- [Possible Code String](other/Possible_Code_strings.py)
```

Comment:
Please keep the change required for this PR only, please remove the rest.

Reply:
Done ma'am ,Made the required changes